dev-python/param/param-1.10.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Make your Python code clearer and more reliable by declaring Parameters"
+HOMEPAGE="
+	http://param.holoviz.org
+	https://pypi.org/project/param
+	https://github.com/holoviz/param"
```

Comment:
Oh, that reminds me. We can add pypi and GitHub as remote-ids in [metadata.xml](https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/) instead.

Reply:
Oh, I forgot about `remote-id`. I'll make that happen everywhere.